vizro-core/docs/pages/user-guides/graph.md

```diff
@@ -84,4 +84,4 @@ import plotly.io as pio
 pio.templates.default = "plotly"
 ```
 
-Alternatively, enter your desired template into any `plotly.express` chart as `template="plotly"` on a case-by-case basis. Note that we do not recommend the above steps for use in dashboards, as other templates will look out-of-sync with our overall dashboard design.
+Or enter your desired template into any `plotly.express` chart as `template="plotly"` on a case-by-case basis. Note that we do not recommend the above steps for use in dashboards, as other templates will look out-of-sync with our overall dashboard design.
```

Comment:
This sounded better to me how it was before. Did vale object to it?

Reply:
Yes 😄 vale wanted to simplify things

Reply:
Maybe "as an alternative" then? Definitely it's not right with "Or" like this though. So I would ignore vale if we can't think of anything better here...

Reply:
I'll just ignore vale then 👍